overlap-scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -19,6 +19,45 @@
 
 # Usage: $0 ($kubectl_flag | --with-kube boolean | -X)* wmw_name
 
+function create_space() { # usage: space_name
```

Comment:
This should be a script on its own, not a function copied into several scripts.

Reply:
Agreed.